### PR TITLE
[FIX] Merge conflict string "=======

### PR DESCRIPTION
**Pull request by rven**
The line Credits fails on the check-merge-conflict for the README.rst file on Merge conflict string "=======

https://github.com/pre-commit/pre-commit-hooks/pull/47/files

CONFLICT_PATTERNS = [
    '<<<<<<< ',
    '=======',
    '>>>>>>> '
]

### CODE REVIEW

**Comment by sbidoul** [comment]
Aw, annoying. @rven Would you open an issue on the pre-commit-hooks project to see if an solution emerges there?

**Comment by rven** [comment]
https://github.com/pre-commit/pre-commit-hooks/issues/426

File: sample_files/pre-commit-13.0/.pre-commit-config.yaml [hunk 1]
```diff
@@ -31,6 +31,8 @@ repos:
   - id: check-docstring-first
   - id: check-executables-have-shebangs
   - id: check-merge-conflict
+    # exclude autogenerated files
+    exclude: /README\.rst$|\.pot?$
```
Comment (asottile) [inline sample_files/pre-commit-13.0/.pre-commit-config.yaml]: this is the suggested solution (in the duplicate to the issue below) fwiw -- there isn't a way to discern between a stray marker and a rst header underline

**Comment by sbidoul** [comment]
Done in #645

**Comment by sbidoul** [comment]
Thanks for reporting!

**Pull request closed without merging**